addons/script-ide/plugin.gd

```diff
@@ -89,15 +101,15 @@ var suppress_settings_sync: bool = false
 #region Enter / Exit -> Plugin setup
 ## Change the Godot script UI and transform into an IDE like UI
 func _enter_tree() -> void:
-	keyword_icon = create_editor_texture(load("res://addons/script-ide/icon/keyword.svg"))
-	func_icon = create_editor_texture(load("res://addons/script-ide/icon/func.svg"))
-	func_get_icon = create_editor_texture(load("res://addons/script-ide/icon/func_get.svg"))
-	func_set_icon = create_editor_texture(load("res://addons/script-ide/icon/func_set.svg"))
-	property_icon = create_editor_texture(load("res://addons/script-ide/icon/property.svg"))
-	export_icon = create_editor_texture(load("res://addons/script-ide/icon/export.svg"))
-	signal_icon = create_editor_texture(load("res://addons/script-ide/icon/signal.svg"))
-	constant_icon = create_editor_texture(load("res://addons/script-ide/icon/constant.svg"))
-	class_icon = create_editor_texture(load("res://addons/script-ide/icon/class.svg"))
+	keyword_icon = create_editor_texture(ICON_KEYWORD)
```

Comment:
This seems to be a safe change. Although I would prefer loading them here, as before.

My reasoning for this was, that we do not really want to preload those icons, as we will never use them. We will use a slightly different texture, made by `create_editor_texture`.

So if you can revert the `load` -> `preload` change while keeping the relative path, I will accept this PR. :)

Reply:
Ah, I see. I mainly used `preload()` to begin with because it allows easy use of relative paths, but since `preload()` is not ideal for this use case, I've now committed a change so that it now uses `load()` again, and concatenates the file path with the script's current base directory. It's _slightly_ less efficient than from back when it used absolute paths, but I think it's probably fine, since this only happens on startup.

```gdscript
# ...
func _enter_tree() -> void:
	var script_path: String = get_script().get_path().get_base_dir()
	
	keyword_icon = create_editor_texture(load(script_path.path_join("icon/keyword.svg")))
	func_icon = create_editor_texture(load(script_path.path_join("icon/func.svg")))
	func_get_icon = create_editor_texture(load(script_path.path_join("icon/func_get.svg")))
	func_set_icon = create_editor_texture(load(script_path.path_join("icon/func_set.svg")))
	property_icon = create_editor_texture(load(script_path.path_join("icon/property.svg")))
	export_icon = create_editor_texture(load(script_path.path_join("icon/export.svg")))
	signal_icon = create_editor_texture(load(script_path.path_join("icon/signal.svg")))
	constant_icon = create_editor_texture(load(script_path.path_join("icon/constant.svg")))
	class_icon = create_editor_texture(load(script_path.path_join("icon/class.svg")))
# ...
```